src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -126,9 +130,12 @@ IObservable<Unit> OnCloneRepository(object state)
                     notificationService.ShowError(Resources.RepositoryCloneFailedNoSelectedRepo);
                     return Observable.Return(Unit.Default);
                 }
+                
                 // The following is a noop if the directory already exists.
                 operatingSystem.Directory.CreateDirectory(BaseRepositoryPath);
-                return cloneService.CloneRepository(repository.CloneUrl, repository.Name, BaseRepositoryPath);
+
+                return cloneService.CloneRepository(repository.CloneUrl, repository.Name, BaseRepositoryPath)
+                    .Do(_ => this.usageTracker.IncrementCloneCount());
```

Comment:
This one is a bit tricky... VS does the cloning, not us, so we fire off the operation and exit, which means we don't yet know if cloning was successful here. Do we care?